vagrant-kubernetes-istio/README.md

```diff
@@ -1,19 +1,103 @@
-# vagrant-kubernetes-istio
+The following instruction details the steps required to run E2E tests with a Vagrant VM on your local machine, so you can test and debug locally.
 
-Set up Kubernetes on VM with vagrant for Istio e2e tests, so that we can run Istio e2e tests on local machine [issue](https://github.com/istio/istio/issues/4536)
+# Benefits:
+1. Set up a local vagrant VM Environment once and run "make e2e_all" to run E2E tests from your development environment.
```

Comment:
should we add that local_cluster flag too here?

Reply:
Added.